resources/migrations/000_migrations.yaml

```diff
@@ -12919,6 +12919,129 @@ databaseChangeLog:
             tableName: metabase_database
             columnName: details
 
+  # Add created_at to Collection
+  - changeSet:
+      id: v45.00-044
+      author: camsaul
+      comment: Added 0.45.0 -- add created_at to Collection
+      changes:
+        - addColumn:
+            tableName: collection
+            columns:
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  remarks: Timestamp of when this Collection was created.
+
+  # Seed Collection created_at based on the max created at of child objects
+  #
+  # At the time of this writing, the following Models can appear in a Collection:
+  #
+  # - App
+  # - Card
+  # - Dashboard
+  # - Pulse
+  # - Timeline
+  # - NativeQuerySnippet
+  #
+  # For Personal Collections, we can give them a created_at of when their corresponding User joined.
+  - changeSet:
+      id: v45.00-045
+      author: camsaul
+      comment: Added 0.45.0 -- seed Collection created_at with value of oldest item
+      changes:
+        - sql:
+            dbms: postgresql
+            sql: >-
+              UPDATE collection c
+              SET created_at = created_ats.created_at
+              FROM (
+                SELECT min(created_at) AS created_at,
+                collection_id
+                FROM (
+                    SELECT created_at, collection_id
+                    FROM app
+                  UNION ALL
+                    SELECT created_at, collection_id
+                    FROM report_card
+                  UNION ALL
+                    SELECT created_at, collection_id
+                    FROM report_dashboard
+                  UNION ALL
+                    SELECT created_at, collection_id
+                    FROM pulse
+                  UNION ALL
+                    SELECT created_at, collection_id
+                    FROM timeline
+                  UNION ALL
+                    SELECT created_at, collection_id
+                    FROM native_query_snippet
+                  UNION ALL
+                    SELECT u.date_joined, c.id
+                    FROM core_user u
+                    LEFT JOIN collection c
+                      ON c.personal_owner_id = u.id
+                    WHERE c.personal_owner_id IS NOT NULL
```

Comment:
I think this logic might be a little iffy since if a User joins in 2022 and adds a Dashboard created in 2021 to their personal Collection this would give that Collection a `created_at` of 2021 instead of 2022 like we'd expect.